packages/host/app/services/command-service.ts

```diff
@@ -0,0 +1,68 @@
+import Service from '@ember/service';
+
+type Command = 'patch';
+
+class CommandHandler {
+  handler: any;
+  functions: Map<Command, ((arg: any) => void)[]>;
```

Comment:
if you decide you want to support async functions, then make sure that this is `(arg: any) => Promise<void>` type for your map value